common/src/main/java/com/wynntils/mc/mixin/ClientPacketListenerMixin.java

```diff
@@ -196,4 +197,9 @@ private void onDisconnectPre(Component reason, CallbackInfo ci) {
         // Unexpected disconnect
         EventFactory.onDisconnect();
     }
+
+    @Inject(method = "handleParticleEvent", at = @At("HEAD"), cancellable = true)
+    private void onParticle(ClientboundLevelParticlesPacket packet, CallbackInfo ci) {
```

Comment:
I assume you're going to do this properly?

Are there some particles that is problematic in the character selection screen? Do Wynncraft generate particles in the container UI and it is showing through? (Just curious why this is needed)

Reply:
Ohh lol, why did this commit. I was testing smth, Wynn has a kick where it constantly kicks people because of particle packets. I will revert this though...